src/dbg.h

```diff
@@ -6,8 +6,8 @@ extern char *prog_name;
 
 void print_version(int quit);
 void print_usage(void);
-void print_usage_error(const char *format, ...) __attribute__((format(printf, 1, 2)));
-void print_error(const char *format, ...) __attribute__((format(printf, 1, 2)));
+void print_usage_error(const char *format, ...);
+void print_error(const char *format, ...);
```

Comment:
I don't understand - if the compiler can't see the format attribute in the declaration, how it can produce meaningful warnings??

IMHO, the macro should be left in the header file.

Reply:
It definitely does work, but I’ll move it to the header and test again on the less common systems later today.  I’ll mark as draft until I get to that.

Reply:
Made the change, I’ll test again tonight.